src/_guides/libraries/private-files.md

```diff
@@ -103,7 +103,7 @@ similar to Ruby's `Gemfile.lock`.
 Regenerating the `pubspec.lock` file lets you test your package
 against the latest compatible versions of its dependencies.
 
-**For application packages**, we recommend that you
-**commit** the `pubspec.lock` file.
-Saving `pubspec.lock` ensures that everyone working on the app
-uses the exact same versions.
+**For application packages**, we recommend that you commit
+the `pubspec.lock` file.
+Saving `pubspec.lock` will track the dependencies in production or
```

Comment:
The word "saving" here is confusing (it sounds like save as in File>Save, but I think you mean something different?)

Reply:
Did you mean "versioning"?

Reply:
oh, yes, my mistake. Thanks for pointing out and I will fix it.